CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- added: 'forceLightAccountCreate' imports prop to force Light Account creation
```

Comment:
With this change, we now have 3 redundant ways to select the account mode:

1. `loginUiInitialRoute` === "new-light-account" or "login-password-light"
2. `experimentConfig.createAccountType`
3. `forceLightAccountCreate`

It seems like we could probably skip this whole PR and use one of the existing options.

For 1, we could adjust `GettingStartedScene` to route to the correct scene, and for 2 we could adjust `LoginScene` to lie about the light experiment, forcing it to 100% when it passes the `experimentConfig` prop to edge-login-ui-rn.

Ideally we would get rid of 1 and leave only the experiment config, so we don't have *any* redundancy, but that's a task for another day.

Reply:
OK, so the big distinction here is that this selects light account *regardless of the number of existing accounts*. In that case, we can keep it, but we should update the release notes and add a JavaDoc comment explaining that distinction.